app/code/Magento/Catalog/Model/Product/Image/Cache.php

```diff
@@ -127,4 +154,93 @@ protected function processImageData(Product $product, array $imageData, $file)
 
         return $this;
     }
+
+    /**
+     * Get themes in use
+     *
+     * @return array
+     */
+    public function getThemesInUse()
```

Comment:
Please add PHP 7.0 specific type hinting  and return types to any newly introduced methods.

Reply:
added